.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## What changes were made
+
+-
+
+## How to test or check results
+
+- URL or instructions
```

Comment:
What about putting this in a comment or removing it? Think the title is very clear and self explanatory.